tests/integration/security/policy_attachment_only/testdata/authz/gateway-authz.yaml.tmpl

```diff
@@ -0,0 +1,37 @@
+apiVersion: security.istio.io/v1beta1
+kind: RequestAuthentication
+metadata:
+  name: default-{{ .To.ServiceName }}
+spec:
+  targetRef:
+    kind: Gateway
+    group: gateway.networking.k8s.io
+    name: {{ .To.ServiceName }}-gateway
+  jwtRules:
+  - issuer: "test-issuer-1@istio.io"
+    jwksUri: "https://raw.githubusercontent.com/istio/istio/master/tests/common/jwt/jwks.json"
+  - issuer: "test-issuer-2@istio.io"
+    jwksUri: "https://raw.githubusercontent.com/istio/istio/master/tests/common/jwt/jwks.json"
+--- 
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: authz-gateway-{{ .To.ServiceName }}
+spec:
+  targetRef:
+    name: {{ .To.ServiceName }}-gateway
+    kind: Gateway
+    group: gateway.networking.k8s.io
+  action: ALLOW
+  rules:
+  - to:
+    - operation:
+        hosts:
+{{- range $svc := .Services }}
+        - "example.{{ $svc.ServiceName }}.com"
+{{- end }}
+    from:
+    - source:
+        requestPrincipals: ["test-issuer-1@istio.io/sub-1"]
+    - source:
+        requestPrincipals: ["test-issuer-3@istio.io/sub-1"]
```

Comment:
What is the significance of test-issuer-3?

Reply:
it isn't used so I will remove it